framework_tvb/tvb/core/services/burst_service.py

```diff
@@ -315,7 +315,7 @@ def prepare_metrics_operation(operation):
         range_values = operation.range_values
         metric_algo = dao.get_algorithm_by_module(MEASURE_METRICS_MODULE, MEASURE_METRICS_CLASS)
 
-        metric_operation = Operation(operation.fk_launched_by, operation.fk_launched_in, metric_algo.id, '',
+        metric_operation = Operation(None, operation.fk_launched_by, operation.fk_launched_in, metric_algo.id,
```

Comment:
We need a `ViewModel` create here also, right ?

Reply:
I just saw, that the VM gets populated in `HPCSchedulerClient._handle_metric_results`, so we are all good here.
Still I think the TODO in this method should be considered in a later refactor